packages/tempus/src/Tempus_StepperIMEX_RK_decl.hpp

```diff
@@ -326,6 +326,10 @@ class StepperIMEX_RK : virtual public Tempus::StepperImplicit<Scalar>,
 
   /// \name Basic stepper methods
   //@{
+    /// Returns the explicit tableau!
+    virtual Teuchos::RCP<const RKButcherTableau<Scalar> > getTableau() const
+    { return explicitTableau_; }
+
```

Comment:
This is only getting the ERK tableau. And what about the DIRK tableau?

Reply:
Yeah, I had to decide what would getTableau() do, since it is in the base class and only returns a single tableau.  Would you like to have a getExplicitTableau() and a getImplicitTableau()?

Reply:
If we can have the base class take a `bool explicit = true`, I think we can keep a single method.

Reply:
Discussed with @sconde and going with getExplicitTableau() and getImplicitTableau() for IMEX and IMEX Partition.